packages/blob-storage/internal/logic/config.go

```diff
@@ -0,0 +1,94 @@
+package logic
+
+import (
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+// CallbackFunc represents the type of the callback function for handling events.
+type CallbackFunc func(string, string, string, types.Log)
+
+// IndexedEvent struct represents the configuration for an indexed event.
+type IndexedEvent struct {
+	Contract  string
+	EventName string
+	EventHash string
+	Callback  CallbackFunc
+}
+
+// NetworkConfig struct represents the configuration for a network.
+type NetworkConfig struct {
+	RPCURL        string
+	BeaconURL     string // Add this field
+	NetworkName   string
+	IndexedEvents []IndexedEvent
+}
+
+// MongoDBConfig holds the configuration for MongoDB.
+type MongoDBConfig struct {
+	Host     string
+	Port     int
+	Username string
+	Password string
+	Database string
+}
+
+// ServerConfig struct represents the configuration for the api server.
+type ServerConfig struct {
+	Port string
+}
+
+// Config struct holds the overall configuration for the application.
+type Config struct {
+	Networks []NetworkConfig
+	MongoDB  MongoDBConfig
+	Server   ServerConfig
+}
+
+// GetConfig loads the configuration from environment variables or a config file.
+func GetConfig() (*Config, error) {
+	cfg := &Config{}
+	cfg.Networks = []NetworkConfig{
+		{
+			RPCURL:      "wss://l1ws.internal.taiko.xyz",
```

Comment:
Do these configuration items don't need to be changed?

Reply:
I think it can change but rare occasions, which would just mean to restart the application (if changed - if ever).

Reply:
I don't think this is a wise move. Once the URL changed you have to modify the code.

Reply:
Yeah, i dont have a strong opinion, but the URL for example is used once, when you initialize the indexer, so.. how you can modify the indexer on the go ? I think you initialize it, and once changes you stop and restart with a new, no ?
But i'd leave it for you guys.